src/getting_started/development/03_wsl_vscode.md

```diff
@@ -71,6 +71,26 @@ Processor Design Team**.
 
 ---
 
+## Upgrading the Ubuntu Version
+By default, WSL only has LTS (Long-term support) versions available.
```

Comment:
Can you add why this is bad? I think it's nice to give contect.

Reply:
I added a link to ubuntu's page on LTS and added a small blurb about needing more up to date packages.